packages/nyc-test-coverage/src/index.ts

```diff
@@ -1,10 +1,52 @@
-import { InjectedSinks } from '@temporalio/worker';
+import { InjectedSinks, BundleOptions, WorkerOptions } from '@temporalio/worker';
 import { CoverageSinks } from './sinks';
 import libCoverage from 'istanbul-lib-coverage';
 
+// Pull `webpack.Configuration` type without needing to import Webpack
+type WebpackConfigType = ReturnType<NonNullable<BundleOptions['webpackConfigHook']>>;
+
 export class WorkflowCoverage {
   coverageMap = libCoverage.createCoverageMap();
 
+  /**
+   * Add all necessary coverage-specific logic to Worker config:
+   * interceptors, sinks, and Webpack config hook.
+   */
+  augmentWorkerOptions(
```

Comment:
Should we also have a similar `augmentBundleOptions()` method to support the `bundleWorkflowCode()` case? I doubt this will be a very common use case, I see a few reasons someone might want to go that way, and it seems trivial to add...

Reply:
+1